db/schema.sql

```diff
@@ -0,0 +1,31 @@
+
+DROP DATABASE IF EXISTS employee_db;
+CREATE DATABASE employee_db;
+
+use employee_db;
+
+CREATE TABLE DEPARTMENT(
+    id INT NOT NULL,
```

Comment:
To ensure the id is correct, you need to add `AUTO_INCREMENT` to each id in the schema.
```suggestion
    id INT NOT NULL AUTO_INCREMENT,
```